cfme/tests/configure/test_zones.py

```diff
@@ -52,7 +51,7 @@ def test_zone_add_cancel_validation():
 @pytest.mark.tier(2)
 @pytest.mark.sauce
 @pytest.mark.meta(blockers=[1216224])
-def test_zone_change_appliance_zone(request):
+def test_zone_change_appliance_zone(request, appliance):
     """ Tests that an appliance can be changed to another Zone """
     zc = current_appliance.get(ZoneCollection)
```

Comment:
**REQUIRED:** As you added ```appliance``` fixture in here, let's remove the calls to ```current_appliance``` too

Reply:
Done